modules/nw-egress-ips-node.adoc

```diff
@@ -10,14 +10,20 @@ You can apply the `k8s.ovn.org/egress-assignable=""` label to a node in your clu
 
 .Prerequisites
 
+ifndef::openshift-rosa[]
 * Install the OpenShift CLI (`oc`).
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+* Install the ROSA CLI (`rosa`).
+endif::openshift-rosa[]
 * Log in to the cluster as a cluster administrator.
 
 .Procedure
 
 * To label a node so that it can host one or more egress IP addresses, enter the following command:
 +
 [source,terminal]
+ifndef::openshift-rosa[]
```

Comment:
Pretty sure this works, but I suggest opening the ifndef above [source,terminal].